js/server_handle.js

```diff
@@ -43,7 +48,7 @@ function saveGame(){
 	}
 	
 	// Retrieving information from PHP file
-	xmlhttp.open("GET", "savegame.php?id=" + userID + "&level=" + level + "&exp=" + experience + "&exp_tnl=" + experience_tnl + "&health=" + health + "&strength=" + strength + "&dex=" + dexterity + "&intellect=" + intellect + "&statpoints=" + stat_points + "&maxstatpoints=" + max_stat_points + "&need_id=" + need_save_ID, true);
+	xmlhttp.open("GET", "savegame.php?id=" + userID + "&level=" + level + "&exp=" +  experience + "&exp_tnl=" + experience_tnl + "&health=" + health + "&strength=" + strength + "&dex=" + dexterity + "&intellect=" + intellect + "&statpoints=" + stat_points + "&maxstatpoints=" + max_stat_points + "&need_id=" + need_save_ID, true);
```

Comment:
Extra space before the variable 'experience'. Won't cause issue with code though.